sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/feature_store_role_assignments.json

```diff
@@ -102,20 +102,34 @@
     "resources": [
```

Comment:
what is the use of this json file, while the workspace_base json also has similar logic?

Reply:
This is used to update role assignment when performing feature store updates, it's possible that either materialization identity, offline store, online store could be updated or not updated.